README.md

```diff
@@ -12,28 +12,61 @@ Enables AWS Config and adds managed config rules with good defaults.
 
 * approved-amis-by-tag: Checks whether running instances are using specified AMIs.
 
+### Backup Plan
+
+* backup-plan-min-frequency-and-min-retention-check: Checks if a backup plan has a backup rule that satisfies the required frequency and retention period. *No* *GOVcloud*
```

Comment:
What does `No GOVcloud` mean exactly?

Reply:
Unfortunately some of these rules are unsupported in certain regions. Example: 

> AWS Region: All supported AWS regions except China (Beijing), China (Ningxia), AWS GovCloud (US-East), AWS GovCloud (US-West), Asia Pacific (Jakarta), Asia Pacific (Osaka) Region

So while we can use them in our dev environments they may not transfer over to our prod. Since not all of our projects are in govcloud, I elected to keep them in this addition.

Reply:
Adding a hyperlink to the docs where that is explained would be helpful.

Reply:
hyperlinks added.